sites/partners/styles/overrides.scss

```
@@ -25,3 +25,10 @@
   // Table row hover background color
   --ag-row-hover-color: #f2f2f2;
 }
+
+Button.button.is-primary {
```

Does it work if we unnest this? 

```suggestion
Button.button.is-primary:hover { 
```

Similar comment for the other file.

---

Yes, that works